lib/firehose/rack/consumer/http_long_poll.rb

```diff
@@ -149,7 +159,7 @@ def handle_request(request, env)
             subscriptions = Consumer.multiplex_subscriptions(request)
             log_request request, subscriptions, env
             subscriptions.each do |sub|
-              respond_async(sub[:channel], sub[:message_sequence], env)
+              respond_async(sub[:channel], sub[:last_message_sequence] || sub[:message_sequence], sub[:params], env)
```

Comment:
There seems to be a disconnect between using `sub[:last_message_sequence] || sub[:message_sequence]` here and just `params.delete('last_message_sequence').to_i` [above](https://github.com/firehoseio/firehose/pull/67/files#diff-b742558035f57063d120660484e09798R135).

Wouldn't it be better to use [`last_message_sequence(sub)`](https://github.com/firehoseio/firehose/pull/67/files#diff-5ac0e6a01a35de99dc98a0fee395123bR10) on this line?